CISS/transforms/CISS_Patient_Value_Map.py

```diff
@@ -0,0 +1,115 @@
+import Global_Constants
+
+ciss_patient_col_specific_value_maps = {
+    'AGE': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'0': 'Less than one year old', '999': 'Unknown'}
+        }
+    ],
+    'SEX': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'3': 'Female, pregnant - 1st trimester (1st-3rd month)',
+                        '4': 'Female, pregnant - 2nd trimester (4th-6th month)',
+                        '5': 'Female, pregnant - 3rd trimester (7th-9th month)',
+                        '6': 'Female, pregnant - trimester unknown'} | Global_Constants.sex_shared_mapping
+        }
+    ],
+    'HEIGHT': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'999': 'Unknown'}
+        }
+    ],
+    'WEIGHT': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': Global_Constants.weight_shared_mapping
+        }
+    ],
+    'ROLE': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': Global_Constants.role_shared_mapping
+        }
+    ],
+    'SEATLOC': [
+        {
+            'ranges': Global_Constants.common_ciss_year_range,
+            'mapping': {'11': 'Front Left', '12': 'Front middle', '13': 'Front right', '14': 'Front other',
```

Comment:
This might be nitpicky but do we want to match case for these? "Front left" instead of "Front Left" to match the others

Reply:
Good call, I think that makes sense to match the case and will make the change